<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_92ceni9a5f/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -16,8 +16,6 @@ build:
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler("c") }}
     - {{ stdlib("c") }}
   host:
@@ -60,7 +58,7 @@ about:
     This standalone variant draws its fixes/improvements from python-trunk
     and provides additional bug fixes and improvements.
   dev_url: https://github.com/celery/billiard
-  doc_ur: https://billiard.readthedocs.io
+  doc_url: https://billiard.readthedocs.io
 
 extra:
   recipe-maintainers:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -22,6 +22,7 @@ requirements:
     - {{ stdlib("c") }}
   host:
     - python
+    - setuptools
     - wheel
     - pip
   run:

